src/lang/std/sketchcombos.ts

```diff
@@ -1441,6 +1493,51 @@ export function removeSingleConstraint({
       // So we should update the call expression to use the inputs, except for
       // the inputDetails, input where we should use the rawValue(s)
 
+      if (inputToReplace.type === 'labeledArg') {
+        if (callExp.node.type !== 'CallExpressionKw') {
+          return new Error(
+            'ADAM: This code path only works with callExpressionKw but a positional call was somehow passed'
```

Comment:
This one is probably fine to keep.

```suggestion
            'This code path only works with callExpressionKw but a positional call was somehow passed'
```